src/datatable/expr/reduce.py

```diff
@@ -110,13 +106,6 @@ def corr(col1, col2):
     return Expr(OpCodes.CORR, (col1, col2))
 
 
-# noinspection PyShadowingBuiltins
-def sum(iterable, start=0):
-    if isinstance(iterable, (Expr, core.FExpr)):
-        return Expr(OpCodes.SUM, (iterable,))
-    else:
-        return _builtin_sum(iterable, start)
-
```

Comment:
This function allowed to use the python `sum()` along with the datatable one. Now, as you removed it, the python sum (in some cases) could be shadowed by the datatable one. For instance, if now someone does `from datatable import *`, the built in `sum()` will be overwritten by datatable version.

Reply:
@oleksiyskononenko any suggestions on this?

Reply:
Just do not remove this shadowing prevention and use `core.sum()` instead of the old reducer function name. Take a look at the changes I just pushed.

Reply:
thanks @oleksiyskononenko